docs/src/components/ComponentDoc/PerfChart/PerfChart.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import { FlexibleXYPlot, XAxis, YAxis, HorizontalGridLines, LineSeries, Crosshair } from 'react-vis'
+
+const ChartTooltip = ({ x, data, ...rest }: { x: number; data: any }) => {
+  return (
+    <Crosshair {...rest} values={[{ x, y: 20 }]}>
+      <div style={{ background: '#555', color: 'white', padding: '.5em' }}>
+        <div>Build:&nbsp;{x}</div>
+        <div>Date:&nbsp;{data.ts}</div>
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/56564694-50c4b500-65b7-11e9-9f77-7574a16448f2.png)
`Date` is missing :(

Reply:
![image](https://user-images.githubusercontent.com/14183168/56565661-96827d00-65b9-11e9-8d35-5adfd15ae0f2.png)

It will be also great to avoid this block in a separate PR...

Reply:
Fixed on service side